### DISCUSSION TITLE
Allow voucher customers to access dashboard when registration disabled

### DISCUSSION THREAD

**Discussion by saifulmd0** [hotspotbilling/phpnuxbill#281] [Ideas]
Hi there,
We disabled manual registration. and we use pre-generated vouchers and print physical voucher codes for customers. Customers can input any username/mobile number and voucher code (as a password) to login to the hotspot directly. It works great!

![image](https://github.com/user-attachments/assets/a7ef39c5-4780-44fa-a668-4a73db2f40fb)

However, It would be very useful if voucher customers could also access the login page via a different login URL/page or something. So they can use the dashboard/portal to buy and renew their vouchers by themselves. They won't have to buy physical cards every time. 

![image](https://github.com/user-attachments/assets/7dbbaa57-ea51-45cc-883d-42b818b6dad2)

Thank you.

**Comment by BiD1** [comment]
saiful vai aper number ta den

**Reply by saifulmd0** [reply]
You can email me at hello[at]saifulislam[dot]info. 

Thanks.

**Comment by saifulmd0** [comment]
I already custom-coded this feature for myself, so I'm going to close this conversation.

**Comment by BiD1** [comment]
ভাই আপনার কাষ্টমাইড টা কি দেখা যাবে
আপনার ফোন নাম্বার টা দিলে কথা বলতাম। আমি এডভান্স হর্টপট টাও ইউজ করি। আমি
বিকাশ আর SSL Commerz ব্যবহার করি

**Reply by saifulmd0** [reply]
আমার কাস্টামাইডজ টা আমার প্রয়োজন অনুযায়ী করা আছে। তাছাড়া নতুন ভার্সন আপডেট দিয়ে আামার কাস্টমাইজ চলে যায়, পরে আবার কোড মডিফাই করে দিতে হয়। 

তারপরও লগিন এর ফাইলটা দিলাম এটা রিপ্লেস করে দিলে ভাউচার এক্টিভেশন আর লগিন দুইটা অপশনই একসাথে দেখাবে। 

Unzip and replace this file in "ui > user-ui >" folder. 
[login-noreg.zip](https://github.com/user-attachments/files/17084095/login-noreg.zip)

**Comment by BiD1** [comment]
ধন্যবাদ ভাই আপনাকে।

**Reply by saifulmd0** [reply]
আপনার এডভান্স হটস্পট টা কিভাবে সেটাপ করেছেন?

**Comment by BiD1** [comment]
পেমেন্ট প্লাগইনের মতো আলাদা প্লাগইন বানাতে হয়েছে ভাই।

**Reply by saifulmd0** [reply]
প্লাগইনটা সেয়ার করা যাবে ভাই?
